data/large/ground-truth/CO2_measured_spectrum_4-5um.json

```diff
@@ -0,0 +1,15 @@
+{
+  "fileName": "CO2_measured_spectrum_4-5um.spec",
+  "molecule": "CO2",
+  "isotope": "1,2",
+  "wmin": 4167,
+  "wmax": 4180,
+  "wunit": "nm",
+  "pressure": 1e-3,
```

Comment:
Note  : does it work if you use Astropy.units there?

Reply:
Actually no, but I can add something like this:

- If the `wunit` is either `"nm"` or `"cm-1"`, then we parse it directly into `calc_spectrum()`.
- If the unit is something else (for example, `km` or `mm` or so), I will convert it into Astropy.units and parse them into `calc_spectrum()` using [Astropy units conversion](https://github.com/radis/radis/blob/b853a26b34a0b9b53670062a2214da4d134ec391/radis/phys/units_astropy.py#L10).

Does it sound good, sir?

Reply:
That would add a new layer and new code. Normally all your inputs are already parsed correctly by calc_spectrum if they are Astropy.units.

Try?

Reply:
Mr. @erwanp , I have tried with other units aside `cm-1` and `nm`, and actually it works. It seems like RADIS automatically tries to convert `wunit` into Astropy.units if the users input some unusual units.

So, to answer your initial question, yes, it works with Astropy.units too.